compiler/modules/nback.llvm/llvm.bal

```diff
@@ -554,6 +573,22 @@ function sameIntType(Value v1, Value v2) returns IntType {
     panic error("expected an int type");
 }
 
+function largerType(IntType t1, IntType t2) returns IntType {
```

Comment:
This is too ugly and too much code.  I don't think it's necessary to check this here. LLVM catches this sort of thing anyway.

Reply:
Fixed